azure-pipelines.yml

```diff
@@ -0,0 +1,297 @@
+# Variable 'keys.ipstack_auth' was defined in the Variables tab
+# Variable 'keys.maxmind_auth' was defined in the Variables tab
+# Variable 'PIP_CACHE_DIR' was defined in the Variables tab
+# Variable 'python.version' was defined in the Variables tab
+# Cron Schedules have been converted using UTC Time Zone and may need to be updated for your location
+# Multi-job configuration must be converted to matrix strategy: https://docs.microsoft.com/en-us/azure/devops/pipelines/process/phases?view=azure-devops&tabs=yaml#multi-job-configuration
+trigger:
+  branches:
+    include:
+    - main
+  batch: True
+pr:
+  branches:
+    include:
+    - main
+schedules:
+- cron: 0 4 * * 1
+  branches:
+    include:
+    - main
+  always: true
+name: $(date:yyyyMMdd)$(rev:.r)
+resources:
+  repositories:
+  - repository: self
+    type: git
+    ref: main
+stages:
+- stage: PythonTests
+  displayName: Python build and tests
+  variables:
+    # This is run explicity so does not need to be auto-injected
+    skipComponentGovernanceDetection: true
+  jobs:
+  - job: InstallAndTestPackage
+    strategy:
+      matrix:
+        linux:
+          imageName: "ubuntu-latest"
+          python.version: '3.6'
+        mac:
+          imageName: "macos-latest"
+          python.version: '3.6'
+        windows-py36:
+          imageName: "windows-latest"
```

Comment:
Do we want to add windows 3.8?

Reply:
We could. I think I remember a funny issue with bokeh on Win + 3.8 that caused a test to fail because of missing attribute.